packages/broker/src/plugins/operator/closeExpiredFlags.ts

```diff
@@ -0,0 +1,33 @@
+import { EthereumAddress, Logger } from '@streamr/utils'
+import { ContractFacade, SponsorshipResult } from './ContractFacade'
+
+const logger = new Logger(module)
+
+export const closeExpiredFlags = async (
+    maxFlagAgeSec: number,
+    operatorContractAddress: EthereumAddress,
+    contractFacade: ContractFacade
+): Promise<void> => {
+    logger.info('running closeExpiredFlags')
+
+    const minFlagStartTime = Math.floor(Date.now() / 1000) - maxFlagAgeSec
+
+    const sponsorships = (await contractFacade.getSponsorshipsOfOperator(operatorContractAddress))
+        .map((sponsorship: SponsorshipResult) => sponsorship.sponsorshipAddress)
+    if (sponsorships.length === 0) {
+        logger.info('no sponsorships found')
+        return
+    }
+    logger.info(`found ${sponsorships.length} sponsorships`)
+    const flags = await contractFacade.getExpiredFlags(sponsorships, maxFlagAgeSec)
+    logger.info(`found ${flags.length} flags`)
```

Comment:
Again would go for `debug` level here and add "expired" as adjective.
```suggestion
    const flags = await contractFacade.getExpiredFlags(sponsorships, maxFlagAgeSec)
    logger.debug(`Found ${flags.length} expired flags`)
```